wcfsetup/install/files/lib/system/form/builder/Psr15DialogForm.class.php

```diff
@@ -44,7 +51,10 @@ public function validatePsr7Request(ServerRequestInterface $request): ?JsonRespo
         return null;
     }
 
-    public function toJsonResponse(): JsonResponse
+    /**
+     * Returns a response that can be consumed by JavaScript's `dialogFactory().usingFormBuilder()`.
+     */
+    public function toJsonResponse(): ResponseInterface
```

Comment:
Perhaps the methods should even be renamed to:

- validateRequest()

and

- toResponse()

respectively?

Reply:
I have no strong preference here, I’m fine with the proposed names.